test/requiregSpec.js

```diff
@@ -52,15 +54,15 @@ describe('requireg', function () {
 
       it('should have the expected module path', function () {
         expect(requiregModule.resolve('beaker'))
-          .to.be.equal(__dirname + '/fixtures/lib/node_modules/beaker/index.js')
-      }) 
+          .to.be.equal(path.join(__dirname, 'fixtures', 'lib', 'node_modules', 'beaker', 'index.js'))
```

Comment:
Yeah I know, but I prefer not to rely on `path.join` normalizing things. Since it normalizes after joining, you temporarily have a path that has potentially both separators and I'm always worried something might get mixed up. I agree it's more a matter of personal taste.

In this case, there was no `path.join` before, and I actually leverage the individual parameters in other tests by passing different values depending on the platform, so it's also more consistent to have it everywhere.